src/test/java/com/github/reducktion/socrates/validator/GermanyIdValidatorTest.java

```diff
@@ -0,0 +1,75 @@
+package com.github.reducktion.socrates.validator;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class GermanyIdValidatorTest {
+    private GermanyIdValidator germanyIdValidator;
+
+    @BeforeEach
+    void setup() {
+        germanyIdValidator = new GermanyIdValidator();
+    }
+
+    /**
+     * These valid numbers are taken from the official document,
+     * mentioned in the JavaDoc of $GermanyIdValidator#validate.
```

Comment:
To link a method in Javadoc, please use `@link`.

```suggestion
     * mentioned in the JavaDoc of {@link GermanyIdValidator#validate}.
```